Autopilot/Inc/Interchip_A.h

```diff
@@ -10,12 +10,14 @@
 #ifdef __cplusplus
 extern "C" {
 #endif
+ 
 
+void Interchip_Init();
 void Interchip_Run();
-int16_t *Interchip_GetPWM(void);
-void Interchip_SetPWM(int16_t *data);
+void Interchip_SetPWM(int index,int data);
 uint16_t Interchip_GetAutonomousLevel(void);
```

Comment:
Each of these will need a comment to explain it. As we do with all our function prototypes.